### ISSUE TITLE
Interim UX Fix Proposal to the Observation Detail Screen

### ISSUE THREAD

**Issue by joellebel** [inaturalist/inaturalist#921]
After watching more folks struggle with the community vote section... 
![image](https://cloud.githubusercontent.com/assets/11527994/13687332/77203d24-e6cf-11e5-84df-3f2908ceed36.png)

...I thought I'd make a few recos. Also because I remember @kueda talking about possibly removing this section from the ob detail screen a few months back. Maybe now's the right time? People seemed to not understand this section's significance, especially in relation to the page heading and the fact that this is not with the ID section of the page.
### I'm proposing the following:

![image](https://cloud.githubusercontent.com/assets/11527994/13687313/59ea0cbc-e6cf-11e5-9249-7bb4ae2619ca.png)

Here's the two side-by side so you can see the difference.

![image](https://cloud.githubusercontent.com/assets/11527994/13688816/24899e7a-e6d9-11e5-8137-d7c18f77cfd5.png)

or a link for comparison: http://www.inaturalist.org/observations/2753218 
1. Remove the voting section because it's clear what the community ID by the page title.
2. Move user info, observe and add date to the right rail
3. Add observer's ob count
4. Move identotron button closer to where users make identifications 
5. The endangered species tag (or any other tag) would go next to the page title.
6. Remove "Back to **Username** Observations" link in the upper left because you can reach the user's page from their ob count

I think the top priority is the voting section, but would be happy if any of the other items were a quick and easy change.

**Comment by loarie** [comment]
I'm not a big fan of that Observation summary view in the upper right, but I suspect some might be (@kueda ?)
But some things that need to be accounted for:
1) I really find the default taxon thumbnail associated with the community ID. And remember the community ID won't always be represented in the observations. Have you thought about including that in the community ID title?
2) There's controls for opting in and out of the community ID in that upper right hand bit and theres going to be the opt-out of community ID case which looks like this http://www.inaturalist.org/observations/2753758 where does that stuff go?
<img width="305" alt="screen shot 2016-03-10 at 4 13 13 pm" src="https://cloud.githubusercontent.com/assets/598026/13689086/033b2c50-e6db-11e5-9960-629cfec305c4.png">

**Comment by loarie** [comment]
Also - we discussed the idea of having the taxon name at the top be 'Unknown' when there is no obs.taxon and showing the 'placeholder' some other way. To me thats a top priority and would be nice to see included in these proposals

**Comment by loarie** [comment]
Also, have you considered moving Observation Fields over to the sidebar (since its sort of related to projects), its annoying that in its current position it can push the comment/id UI down so far
<img width="1026" alt="screen shot 2016-03-10 at 4 20 37 pm" src="https://cloud.githubusercontent.com/assets/598026/13689194/10e2c768-e6dc-11e5-9f6b-998214a9ad75.png">

**Comment by naturalistcharlie** [comment]
I'd be sad to lose it since I use that section to verify id. But it's not a
big deal.

On Thursday, March 10, 2016, Scott Loarie <notifications@github.com> wrote:

> Also, have you considered moving Observation Fields over to the sidebar
> (since its sort of related to projects), its annoying that in its current
> position it can push the comment/id UI down so far
> [image: screen shot 2016-03-10 at 4 20 37 pm]
> <https://cloud.githubusercontent.com/assets/598026/13689194/10e2c768-e6dc-11e5-9f6b-998214a9ad75.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inaturalist/inaturalist/issues/921#issuecomment-195113633>
> .
>


-- 
Sent from Gmail Mobile

**Comment by joellebel** [comment]
> 1) I really find the default taxon thumbnail associated with the community ID. And remember the community ID won't always be represented in the observations. Have you thought about including that in the community ID title?

I was trying to make as few changes as possible, but think this is a great idea. I would just put it a little lower on the priority list. 

> 2) There's controls for opting in and out of the community ID in that upper right hand bit and theres going to be the opt-out of community ID case which looks like this http://www.inaturalist.org/observations/2753758 where does that stuff go?

Thanks for pointing this out. Here's a solution...
### Opt Out Question
Move that below your own ID since the user's eyeballs should be in that area after making their ID.

![image](https://cloud.githubusercontent.com/assets/11527994/13689508/2ac37838-e6de-11e5-9fb9-b65092bd7ae4.png)


### Opt Out State 

This is how it would look for your own ob. If it's someone else's ob, it's that messaging, but in the same place. 

![image](https://cloud.githubusercontent.com/assets/11527994/13689704/a61a1040-e6df-11e5-9c05-3311e92d4352.png)


> Also, have you considered moving Observation Fields over to the sidebar (since its sort of related to projects), its annoying that in its current position it can push the comment/id UI down so far

Not a bad idea, but would like to hold off on that for a later day. It doesn't seem as much of a problem. In the meantime we could make it expandable/collapsable like we've done to the location and data quality assessment panels. 


> Also - we discussed the idea of having the taxon name at the top be 'Unknown' when there is no obs.taxon and showing the 'placeholder' some other way. To me thats a top priority and would be nice to see included in these proposals

Was going to make a separate issue out of that, but we can certainly add it here. Let me get cracking on some options.

**Comment by loarie** [comment]
> 1) I really find the default taxon thumbnail associated with the community ID. And remember the community ID won't always be represented in the observations. Have you thought about including that in the community ID title?

> I was trying to make as few changes as possible, but think this is a great idea. I would just put it a little lower on the priority list. 

My only point was that your proposed changes would remove my ability to see what the community ID is supposed to look like, which is a feature I like, so if others agree I'd prefer not removing it. In any case, not removing an existing thing is different than prioritizing adding a new thing

**Comment by loarie** [comment]
Re: Opt Out Question
If we are going through with removing the ID summary int he upper right (which I'm fine with but curious what others think), I'd rather just remove it rather than fold it in to the comments/ID feed. All I meant was that (a) we need some place for the About/reject link (to the right of the display of the community ID at the very top of the page seems like the natural place to me  (e.g. next to CR)
<img width="696" alt="screen shot 2016-03-10 at 5 03 16 pm" src="https://cloud.githubusercontent.com/assets/598026/13690025/ffda840a-e6e1-11e5-97e0-79bdd351ebf9.png">
seems like we only need to display the 'you've opted out of Community ID' thing if your ID differs from the ID which rarely happens (right now we always show it). And we could also put that in the reject/aout link at the top. I guess seems like these should be associated with the community ID and the only place we show that is at the top according to these proposed changes

**Comment by joellebel** [comment]
# Placeholder 

### Dir 1: In Title
![image](https://cloud.githubusercontent.com/assets/11527994/13690727/b9cbcf0e-e6e7-11e5-80f9-6b9f9fa8f0ff.png)

### Dir 1.1: In Title, Labelled As
![image](https://cloud.githubusercontent.com/assets/11527994/13690948/86ded6ac-e6e9-11e5-8a60-2899e5c7f504.png)



### Dir 1.2: In Title, No Label

![image](https://cloud.githubusercontent.com/assets/11527994/13690731/c1f17bde-e6e7-11e5-9d32-7ea00a36686a.png)

### Dir 1.2: In Title, Quotes

![image](https://cloud.githubusercontent.com/assets/11527994/13690734/cf839d0e-e6e7-11e5-82c8-d8f1391a3c18.png)





### Dir 2: 

This direction doesn't seem as clear, but I thought I'd post it for comparison...

![image](https://cloud.githubusercontent.com/assets/11527994/13690786/4d0bfd52-e6e8-11e5-9e37-aa53a1112097.png)



It could also go below comments & identifications as sort of a notice. Let's see what you guys think of this first. I prefer Direction 1.1 since it's the most explicit.

**Comment by loarie** [comment]
I think I like 1

**Comment by naturalistcharlie** [comment]
for those of us who do a bunch of species ID, this would make it slower to
hit 'agree'. Though I know the ID Please page is getting an overhaul so
maybe that will counteract it.

On Thu, Mar 10, 2016 at 9:07 PM, Scott Loarie <notifications@github.com>
wrote:

> I think I like 1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inaturalist/inaturalist/issues/921#issuecomment-195144035>
> .
>



-- 
============================
Charlie Hohn
Montpelier, Vermont

**Comment by kueda** [comment]
I don't really think any of these treatments improve this page, and I don't like any of the proposals so far to show what the observation's taxon is when I've opted out of the community ID and my ID differs from the community ID. Giving it such a low priority makes me feel like this is no longer my observation. It also doesn't reflect the state of the data: if I disagree with the community ID and have opted out, the observation's taxon is the one *I* chose, and that was very intentional, because we think observers should always have the ability to control the core components of an observation, like the taxon, even if that ability is opt-in.

IMO, the current treatment is a little confusing because you can make IDs in multiple places, but that inconvenience is outweighed by the fact that you have a quick, at-a-glance way to see 

1. What the observer thinks this is
1. What the community thinks this is
1. What alternative taxa have been proposed
1. How much support each alternative taxon has

And as Charlie pointed out, you can quickly agree without scrolling, which might seem trivial unless you're actually one of the people who perform the critical service of identifying stuff and you agree 100s of times a day.

While I agree there is a lot of frustrating design (or lack of design) on the obs detail page, I don't think an incremental approach is in order. This is a central and complicated part of the experience and if it needs to change, I think it deserves some serious requirements gathering and a complete overhaul. I think little "fixes"  like this are going to annoy experienced users more than they help new users.

**Comment by naturalistcharlie** [comment]
fwiw already really missing the ability to add ID in the upper right. It's not a big deal, but it's hard for me to imagine the upsides of this (not sure what they are) outweigh the big downside of making it harder for people to add IDs to observations. I know there is the 'i' shortcut but that isn't going to be immediately apparent to everyone. 
Just my two cents and I will promise to try not to belabor it :)

**Comment by loarie** [comment]
I hear Ken-ichi's point about marginal cost/benefit on the community ID treatment

But I do think the proposed Placeholder changes will have much more benefit than cost. Anyone apposed to these?

**Comment by naturalistcharlie** [comment]
The placeholder thing sounds good to me!

**Comment by joellebel** [comment]
I am not opposed to the Placeholder changes and also think they will have much more benefit than cost. 

Agreed, this page needs a definite overhaul and usually bandaid solutions aren't the greatest. My thinking was that with the possible influx of new users due to the NPS Centennial bioblitzes these interim fixes might outweigh the costs. 

@kueda Awhile back you mentioned a desire to remove the ID summary. Unfortunately, I cannot find a record of this conversation. Did you have an alternative design in mind for the ID Summary? ...or maybe I'm not remembering correctly? Here is the issue I created for this in the design repo back when it was first mentioned. (https://github.com/inaturalist/design/issues/28) Either way, I would love to hear your alternative design (if you had one) since I will be designing the Ob Detail page soon. Tnx!

**Comment by kueda** [comment]
I doubt I expressed such a desire, though who knows, I say all kinds of crazy things. The only thing I don't like about the current state of affairs is that there is duplication between the ID summary and the list of IDs below (with the comments), but I think the summary is performing an important service. I don't have an alternate design in mind, mostly because this is a hard problem without an obvious design solution. The current version of this page *is* my attempt to solve that problem. I suspect an better design would perform the summary requirements I enumerated [above](https://github.com/inaturalist/inaturalist/issues/921#issuecomment-196448444), as well as

1. Show the IDs and comments in one place
1. Show the sequence of IDs and comments in the order they were created, including past IDs that have since been superseded
1. Maybe even show the changing state of the community IDs at different points in the history (something the current design does not do)

Seems like you are all out-voting me on the incremental changes. To reiterate, I think they're all incremental changes for the worse that remove functionality without adding something better, but I guess none of you agree.
